### PR TITLE
fix(LINGUIST-414): Update profile fix

### PR DESCRIPTION
**Pull request by KardelenCeren**
Minor fix to update profile - it was giving duplicate email error while updating and some json fields were broken

### CODE REVIEW

**Review by can-aslan — APPROVED** [review]
LGTM

File: profiling/views/update_user_profile.py
Comment (tolgaozgun) [inline profiling/views/update_user_profile.py:1]: Why is this change required? This file is for user service to update the ml service

File: profiling/models.py
Comment (tolgaozgun) [inline profiling/models.py:1]: Why is this required? We are already using json.loads and json.dumps, the existing DB is not updated, hence your issues.

**Review by tolgaozgun — CHANGES_REQUESTED** [review]
I didn't understand the changes, TextField is currently working.

**Pull request closed without merging**